GCDWebUploader/GCDWebUploader.m

```diff
@@ -296,7 +296,13 @@ @implementation GCDWebUploader
 
 - (instancetype)initWithUploadDirectory:(NSString*)path {
   if ((self = [super init])) {
-    NSBundle* siteBundle = [NSBundle bundleWithPath:[[NSBundle mainBundle] pathForResource:@"GCDWebUploader" ofType:@"bundle"]];
+    NSString *bundlePath = [[NSBundle mainBundle] pathForResource:@"GCDWebUploader"
+                                                           ofType:@"bundle"];
+    if (!bundlePath) {
```

Comment:
Let's say you want to use a custom GCDWebUploader.bundle (change the CSS to match your app design for example) if I force `[[NSBundle bundleForClass:[GCDWebUploader class]]` you won't be able to change it without touching the .framework directory.

With the current design it first searches for the bundle on the application bundle and then on the framework's bundle.

Any way, it could be removed, and maybe later add a more explicit way of setting the `GCDWebUploader.bundle` in case you want to customize it.